### ISSUE TITLE
Skip to content accessibility workarounds no longer work

### ISSUE THREAD

**Issue by sbcatania** [elementor/elementor#17433]
This method originally posted by @jessicaoddi in https://github.com/elementor/elementor/issues/9292#issuecomment-920169301_ no longer works. You'll notice even on this demo site http://streetpatios.ca/?cmp_bypass=preview linked in this answer, it no longer actually skips the menu. It just scrolls down a bit more and then the tab index continues on to the menu rather than skipping it. Elementor needs to fix this, and there should be a default option for this within the Elementor editor it should not take all of this custom code to implement something that should be standard on every website. Currently, I cannot find any way to add skip to content on our site or any site even while getting creative with custom code. It'll just scroll the page to the CSS id you specify but it won't then actually move the tab index to that link.

**Comment by jessicaoddi** [comment]
> Is anyone ever going to make a fix for this? This is an accessibility failure

Seconded! Get on this Elementor we need a solid solution.
